src/server/process_request.c

```diff
@@ -962,12 +925,7 @@ dispatch_request(int sfds, struct batch_request *request)
 			req_register(request);
 			break;
 
-		case PBS_BATCH_AuthenResvPort:
-			if (pbs_conf.auth_method == AUTH_MUNGE) {
-                                req_reject(PBSE_BADCRED, 0, request);
-                                close_client(sfds);
-                                return;
-                        }
+		case PBS_BATCH_Authenticate:
```

Comment:
Now instead of "PBS_BATCH_AuthenResvPort and PBS_BATCH_AuthExternal" we will be using "PBS_BATCH_Authenticate".  Then why are we handling only req_authenResvPort() inside this (PBS_BATCH_Authenticate) ?

Reply:
for other than resvport, it's handled in process_request().